frontend/simple/views/CreateGroup.vue

```diff
@@ -223,18 +221,18 @@ export default {
           founderIdentityContractId: this.$store.state.loggedIn.identityContractId
         })
         const hash = entry.toHash()
-        // TODO: convert this to SBL
-        Vue.events.$once(hash, (contractId, entry) => {
-          this.$store.commit('setCurrentGroupId', hash)
-          // Take them to the invite group members page.
-          this.$router.push({path: '/invite'})
-        })
-        // TODO: convert this to SBL
-        await backend.publishLogEntry(hash, entry)
-        // add to vuex and monitor this contract for updates
+
+        await sbp('transactions/v1/run', 'Create a Group', false, [
+          { execute: 'setInScope', args: { contractId: hash, entry } },
+          { execute: 'backend/v1/publishLogEntry', description: 'Publish Group Contract', args: { contractId: 'contractId', entry: 'entry' } }
+        ])
+
         await this.$store.dispatch('syncContractWithServer', hash)
-      } catch (error) {
-        console.error(error)
+        this.$store.commit('setCurrentGroupId', hash)
+        // Take them to the invite group members page.
+        this.$router.push({path: '/invite'})
```

Comment:
As per previous code, these should only be done upon server echoing back transaction to us and it being successfully processed by `handleEvent`